src/components/ActionBar/src/InlineActionBar.vue

```diff
@@ -21,9 +28,37 @@ import AtomicActionBar from './AtomicActionBar.vue';
 export default {
 	components: {
 		AtomicActionBar,
+		MTransition,
 	},
 
 	inheritAttrs: false,
+
+	props: {
```

Comment:
If we make these public we'll need to add documentation for them. 

In the short-term I'm wondering if we should hold our cards a bit closer to our chest and keep the configurations as internal only. This way we could explore more global configurations across several components and consider how theming might enable different styles of animations.

Reply:
Removed the props from InlineActionbar so that there's just the default transition and delay, similar to ActionBar.